WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelAddNewPackageViewModel.swift

```diff
@@ -5,23 +5,157 @@ import Yosemite
 /// View model for `ShippingLabelAddNewPackage`.
 ///
 final class ShippingLabelAddNewPackageViewModel: ObservableObject {
+    private let stores: StoresManager
+    private let siteID: Int64
+
+    /// Index of the selected tab
+    ///
     @Published var selectedIndex: Int
+
+    /// View for the selected tab
+    ///
     var selectedView: PackageViewType {
         PackageViewType(rawValue: selectedIndex) ?? .customPackage
     }
 
     // View models for child views (tabs)
-    let customPackageVM = ShippingLabelCustomPackageFormViewModel()
-    let servicePackageVM: ShippingLabelServicePackageListViewModel
+    var customPackageVM = ShippingLabelCustomPackageFormViewModel()
+    lazy var servicePackageVM = ShippingLabelServicePackageListViewModel(packagesResponse: packagesResponse)
```

Comment:
Nit: I think these should be `private(set)` to make sure that the variables can only be updated within this view model.